source/shared/net/guides/run-node-macos.rst

```diff
@@ -0,0 +1,370 @@
+
+#. Click **Continue** on the *License* page, and then **Agree** to accept the license agreement.
+
+#. On the *Configuration* page you have the following options for both a
+   mainnet and testnet node.
+
+   - **Run a [Mainnet/Testnet] node at start-up**: When selected, the node runs
+     when the system starts. Choose this option when you plan to use the node
+     frequently and need it to be up-to-date at short notice. If you don’t
+     select this option, you’ll have to start the node manually when required,
+     for example, when you want to use the Desktop Wallet or concordium-client.
+     If you choose to start the node manually, it might take longer for the node
+     to get up-to-date with the blockchain depending on when the node was last
+     up-to-date.
+
+   - **Start the node after installation is complete**: The node will
+     automatically start running after the installation is complete. Do not
+     choose this option if you want to make further configuration changes before
+     starting the node.
+
+   - **Report to the network dashboard**: Select this option if you want to publish your node statistics to the relevant dashboard when the node is running. Go to the mainnet or testnet dashboard to view the statistics:
+
+     - https://dashboard.mainnet.concordium.software/
+
+     - https://dashboard.testnet.concordium.com/
+
+   - **Public node name**: Specify the name of your node as you want it to appear
+     on the network dashboard if the node reports to the network dashboard. *NB:
+     You must enter a name for both nodes, even if you only want to run one of them.*
+
+#. Once configured, click **Continue** to go to the *Installation Type* page.
+
+#. Click **Install** and enter your administrator password. The installer needs
+   the password because it installs the node for all users and runs it as a
+   system service.
+
+#. Once installed, click **Close**. The following helper-applications are now
+   available on your computer (you can view them in the LaunchPad or in the
+   */Applications/Concordium Node* folder):
+
+   - **Concordium Node Start Mainnet**
+
+   - **Concordium Node Stop Mainnet**
+
+   - **Concordium Node Start Testnet**
+
+   - **Concordium Node Stop Testnet**
+
+Verify that the node is running
+===============================
+
+The node runs as a background service with no user interface. To verify that
+it's running, use the **Activity Monitor**.
+
+#. Open the **Activity Monitor** application from the LaunchPad.
+
+#. Look at the list and verify that the following is running:
+
+   - ``concordium-node`` for each node you are running. For example, if you're
+     running a node on both testnet and mainnet, you'll see two instances of
+     concordium-node.
+
+   - ``node-collector`` for each node that reports to a network dashboard.
+
+#. If the node is running properly and reporting to the network dashboard, you
+   can see it on the dashboard:
```

Comment:
```suggestion
#. If the node is running properly with reporting to the network dashboard enabled, you
   can see the name of the node on the dashboard:
```
I think it would be good to emphasize reporting should be enabled for this